distributions/nr-otel-collector/THIRD_PARTY_NOTICES.md

```diff
@@ -21,166 +21,6 @@ t/.
 
 
 
-## [github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter](https://github.com/open-telemetry/opentelemetry-collector-contrib)
```

Comment:
TODO: figure out why these got deleted when running `make licenses`

Reply:
- diff is due to ci generating an empty license file, so the new build process is not yet compatible with ci
- still unclear why my local machine generates a different license file than mailo-nr [on his machine](https://github.com/newrelic/opentelemetry-collector-releases/pull/130/files#diff-e4cb27b3d6a773cd0e9e99ecb1364aaa663f46303065e8e14fb0e8f198c34658) for a very similar change